app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -95,17 +93,15 @@ public EsriGGMapView(Context context, AttributeSet attrs) {
     @Override
     public void unpause() {
         super.unpause();
-        if (mCompass != null) {
-            mCompass.start();
-        }
+        startPlugin(mCompass);
+        startPlugin(mScaleBar);
```

Comment:
I'm not sure it's needed but these can be grouped under "startPlugins(). And the same for the stop...